src/Connection.ts

```diff
@@ -137,8 +136,14 @@ class Connection {
 	) {
 		this.logger = logger;
 		this.cacheMaxAge = cacheMaxAge;
-		this.endpoint = `${mvisUri}/${account}/subroutine/${Connection.getServerProgramName('entry')}`;
-		this.timeout = timeout;
+
+		const baseURL = `${mvisUri}/${account}/subroutine/${Connection.getServerProgramName('entry')}`;
+
+		this.axiosInstance = axios.create({
+			baseURL,
+			timeout,
+			transitional: { clarifyTimeoutError: true },
```

Comment:
Any idea why this is "transitional" and might go away in future versions? Do they expect you to parse the timeout message?

Reply:
I didn't find anything when I originally looked, but now I see this in the Github README:
```
// transitional options for backward compatibility that may be removed in the newer versions
```
So it seems likely that at some stage there will be a breaking change.  That might mean we can't ascertain this anymore -- it might mean there's an API change, it might mean they always send the timeout error in this situation without the flag.  I can't find anything else about the intent on it.  I guess when updating the version some care will need to be exercised.